apps/central-scan/integration-testing/cypress/e2e/configuration.cy.ts

```diff
@@ -104,10 +104,6 @@ describe('BSD and services/Scan', () => {
     cy.contains('No ballots have been scanned', { timeout: 30000 });
     cy.contains('Scan New Batch').click();
 
-    // The ballot package is currently out of sync with the image fixtures
-    // TODO: fix this, probably once our ballot package format changes
-    cy.contains('Wrong Election');
-
```

Comment:
@eventualbuddha are you removing this because you updated the image fixtures? In which case we could check for the correct ballot interpretation.

If not, I do think it's valuable to check this because it tests that the interpreter actually tried to do something.

Reply:
This was actually failing, I think because I updated the fixtures.